docs/source/integrations/lightning_flash.rst

```diff
@@ -0,0 +1,446 @@
+            # 1. Download data
+            download_data("https://pl-flash-data.s3.amazonaws.com/kinetics.zip")
+            
+            # 2. Load data into FiftyOne
+            train_dataset = fo.Dataset.from_dir(
+                "data/kinetics/train",
+                fo.types.VideoClassificationDirectoryTree,
+                label_field="ground_truth",
+                max_samples=5,
+            )
+            
+            val_dataset = fo.Dataset.from_dir(
+                "data/kinetics/val",
+                fo.types.VideoClassificationDirectoryTree,
+                label_field="ground_truth",
+                max_samples=5,
+            )
+            
+            predict_dataset = fo.Dataset.from_dir(
+                "data/kinetics/predict",
+                fo.types.VideoDirectory,
+                max_samples=5,
+            )
+            
+            # 3. Finetune a model
+            classifier = VideoClassifier.load_from_checkpoint(
+                "https://flash-weights.s3.amazonaws.com/video_classification.pt",
+                pretrained=False,
+            )
+            
+            datamodule = VideoClassificationData.from_fiftyone_datasets(
+                train_dataset=train_dataset,
+                val_dataset=val_dataset,
+                predict_dataset=predict_dataset,
+                label_field="ground_truth",
+                batch_size=8,
+                clip_sampler="uniform",
+                clip_duration=1,
+                video_sampler=RandomSampler,
+                decode_audio=False,
+                num_workers=8,
+            )
+            
+            trainer = flash.Trainer(max_epochs=1, fast_dev_run=1)
+            trainer.finetune(classifier, datamodule=datamodule)
+            trainer.save_checkpoint("video_classification.pt")
+            
+            # 4. Predict from checkpoint
+            classifier = VideoClassifier.load_from_checkpoint(
+                "https://flash-weights.s3.amazonaws.com/video_classification.pt",
+                pretrained=False,
+            )
+            
+            classifier.serializer = FiftyOneLabels()
+            
+            filepaths = predict_dataset.values("filepath")
+            predictions = classifier.predict(filepaths)
+            
+            predict_dataset.set_values("predictions", predictions)
+            
+            # 5. Visualize in FiftyOne App
+            session = fo.launch_app(predict_dataset)
+
+
+.. _adding-model-predictions:
+
+Adding model predictions
+________________________
+
+Once you have a trained Flash model, there are a couple of ways that 
+you can use the FiftyOne integrations to
+add generate and add model predictions to your |Dataset| or |DatasetView|.
+
+
+Apply model
+-----------
+
+The easiest way to generate predictions on an existing |Dataset| or |DatasetView| is
+to use the :meth:`apply_model() <fiftyone.core.collections.SampleCollection.apply_model>`
+function, passing in your Flash model.
```

Comment:
I would mention here that, behind the scenes, FiftyOne will construct the appropriate Flash `Trainer()` and use it to perform possibly-distributed inference to generate the predictions.

Speaking of, we should allow `kwargs` to be passed to `apply_model()` that, in the case of Flash models, will be passed to `Trainer(**kwargs)`, so that the user can provide arguments like `num_gpus=32` to configure the distributed inference.